pkg/consumer/consumer.go

```diff
@@ -4,4 +4,5 @@ import "io"
 
 type Consumer interface {
 	Consume(reader io.Reader, destPath string) error
+	EnableOverwrite()
```

Comment:
This shouldn't go on the Consume interface, it should go in configuration options for the consumer when we construct it.